build/nl_server/Dockerfile

```diff
@@ -31,6 +31,13 @@ RUN pip3 install --no-cache-dir -r /workspace/nl_server/requirements.txt
 COPY nl_server/. /workspace/nl_server/
 COPY shared/. /workspace/shared/
 
+# Download nl files from gcs
+COPY deploy/nl/catalog.yaml .
+COPY build/nl_server/requirements.txt /workspace/build/nl_server/requirements.txt
+COPY build/nl_server/download_nl_files.py .
```

Comment:
can you copy these files to keep the git file path structure?

That way, you can use the get_download_root() function and will not break due to files movement in future PRs.

Reply:
sorry, I'm not sure what you mean by copy to keep the git file path structure?

Reply:
I mean 
`COPY build/nl_server/download_nl_files.py /workspace/build/nl_server/download_nl_files.py`

Reply:
hm, I tried making that update but getting an error in the next line when trying to run the file .. I think because download_nl_files.py file needs to be at the same folder level as nl_server